crates/solidity/inputs/language/src/definition.rs

```diff
@@ -90,29 +90,44 @@ codegen_language_macros::compile!(Language(
                         Enum(
                             name = Pragma,
                             variants = [
-                                EnumVariant(reference = AbicoderPragma),
-                                EnumVariant(reference = ExperimentalPragma),
+                                EnumVariant(reference = AbicoderPragma, enabled = From("0.7.5")),
+                                EnumVariant(
+                                    reference = ExperimentalPragma,
+                                    enabled = From("0.4.16")
+                                ),
                                 EnumVariant(reference = VersionPragma)
```

Comment:
nit: given our understanding now, `AbicoderPragma` and `ExperimentalPragma` are much less likely to exist, compared to `VersionPragma`. WDYT of declaring it first? it should be marginally faster to parse for most cases:

```rust
    EnumVariant(reference = VersionPragma),
    EnumVariant(reference = AbicoderPragma, enabled = From("0.7.5")),
    EnumVariant(
        reference = ExperimentalPragma,
        enabled = From("0.4.16")
    )
```